templates/welcome_email.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <p>Hi {{firstName}},</p>
+    <br>
+    <p>Thanks for filling out the form on <a href="https://www.rememberthee.com" target="_blank">rememberthee.com</a>.
+        Remember Thee is available for download on iOS only. <br>You will receive an email from Apple's app testing
+        product, <a href="https://testflight.apple.com" target="_blank">TestFlight</a>, with instructions on how to
+        download Remember Thee shortly.</p>
+
+    <p>A few things to keep in mind while downloading and using Remember Thee:</p>
+    <ul>
+        <li>This version of Remember Thee does not contain all the features you saw on the website.
+            They will be added in the coming days and weeks. If there was a feature you were particularly excited
+            about, please let me know.
+        </li>
+        <li>In general, if there are things you wish the app could do, or if it's not really working for you, please let
+            me know.
+        </li>
+        <li>If the app crashes, something looks visibly broken or doesn't work or is difficult to use, or you're having
+            trouble downloading the app,
+            You can reach out to me either at this email or at help@rememberthee.com, and I'll get back to you as soon
```

Comment:
lower case "y" in "You can reach out...". Also move this up to after "app," in the line above, please